### ISSUE TITLE
wxbtool to facilitate development of pytorch-based models

### ISSUE THREAD

**Issue by mountain** [pangeo-data/WeatherBench#36]
Hi, buddies,

Just for your information, we released a weatherbench toolkit for model development in PyTorch, and yesterday we upgrade the project

* https://github.com/caiyunapp/wxbtool
* https://github.com/caiyunapp/wxbtool/wiki/quick-start

Hope this could help the community.

Mingli

**Comment by mountain** [comment]
And the default demo model is for a 3-day prediction for t850, and can reach a performance of 1.98 which is same level of Weyn et al. 2020 (iterative). Meanwhile we did not introduce complex coordinate transformation and more data from other datasource.

**Comment by raspstephan** [comment]
This is great. Let me know if you get results once the evaluation is fixed. We can put it on the WeatherBench leaderboard.

**Comment by mountain** [comment]
Thanks, I am working on it. Since learning rate need be re-tuned, it cost some time to test all the cases(t850 and z500, 3d/5d).
